### DISCUSSION TITLE
Question about IMU(Inertia measurement unit).

### DISCUSSION THREAD

**Discussion by JaeHoon-An** [raisimTech/raisimLib#146] [Q&A]
Hello!
I'm an undergraduate student at Pusan National University, South Korea.
I have a question about the sensing part.
Can I add any sensors like IMU in Raisim and get the data by simple codes?
Or may I have to calculate from the associated jacobians?

Thanks for inventing this amazing simulator!

**Comment by jhwangbo** [comment]
You can get the angular velocity of a body using ``getAngularVelocity`` method.

You cannot get the linear acceleration of a body. You have to numerically differentiate the linear velocity. You can get a linear velocity of a frame. Please check this link: https://raisim.com/sections/ArticulatedSystem.html#frames

**Reply by JaeHoon-An** [reply]
Thank you for your answers!
I will try this method. :D
